system/automation/expr_types.yaml

```diff
@@ -48,6 +48,28 @@ types:
       - { name: 'archivedAt',      exprType: 'DateTime',   goType: '*time.Time', mode: ro }
       - { name: 'deletedAt',       exprType: 'DateTime',   goType: '*time.Time', mode: ro }
 
+  SinkResponse:
```

Comment:
If we go with HttpRequest and HttpResponse we might manage to have future compatibility here. Maybe?